lua/rust-tools/dap.lua

```diff
@@ -17,7 +22,7 @@ function M.get_codelldb_adapter(codelldb_path, liblldb_path)
   }
 end
 
-function M.setup_adapter()
+local function setup_adapter()
```

Comment:
Why is making it private necessary?

Reply:
It's not. What's the usecase for making it public?

Reply:
1. It already was so without any reason you should keep it the same
2. The user could call setup themselves whenever they want (which btw the loaded check maybe could be inside setup_adapter?)